homeassistant/components/tmb/config_flow.py

```diff
@@ -0,0 +1,99 @@
+"""Config flow for TMB."""
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_NAME
+
+from .const import (
+    CONF_FROM_LATITUDE,
+    CONF_FROM_LONGITUDE,
+    CONF_LINE,
+    CONF_SERVICE,
+    CONF_STOP,
+    CONF_TO_LATITUDE,
+    CONF_TO_LONGITUDE,
+    DOMAIN,
+    SERVICE_IBUS,
+    SERVICE_PLANNER,
+)
+
+
+class TMBConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """TMB config flow."""
+
+    VERSION = 1
+
+    async def async_step_user(self, user_input=None):
+        """Handle a flow initialized by the user."""
+        # Check if user has configured the tmb object on the YAML file
+        if DOMAIN not in self.hass.data:
```

Comment:
You can ask for the credentials via the config flow too.

Reply:
Sorry @balloob , but how can I store this credentials in a secure way? Is there any method to store and retrieve them to/from the database?